### ISSUE TITLE
video classification modelReady callback

### ISSUE THREAD

**Issue by shiffman** [ml5js/ml5-adjacent#8]
@cvalenzuela can you take a look at:
https://github.com/ml5js/ml5-adjacent/tree/master/02_ImageClassification_Video

I couldn't get the `modelReady()` callback to work so just used `setTimeout()`. Also it didn't work with `loadImage()` but that's not a big deal for this. I'm rushing through these so easily could be missing stuff, doing something weird.

**Comment by cvalenzuela** [comment]
Just pushed an update here: https://github.com/ml5js/ml5-adjacent/pull/10 that makes it work. There's a bug (just learned that `typeof null` is `object`¿?!) on parsing the arguments, but this works for now:

```javascript
classifier = ml5.imageClassifier('MobileNet', {}, modelReady);
```

**Comment by shiffman** [comment]
Fixed with #10, thank you @cvalenzuela!
